packages/expo-updates/ios/EXUpdates/Events/NoOpUpdatesEventManager.swift

```diff
@@ -0,0 +1,18 @@
+//  Copyright © 2019 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+internal class NoOpUpdatesEventManager: UpdatesEventManager {
+  internal weak var appContext: AppContext?
+  internal var shouldEmitJsEvents: Bool = false
+
+  internal func setAppContext(appContext: AppContext?) {
+    self.appContext = appContext
+  }
```

Comment:
```suggestion
```


maybe we can remove this explicit `setAppContext` setter in all concrete UpdatesEventManager implementations since we now resolve the previous problem by `AnyObject`.